main.tf

```
@@ -1,5 +1,5 @@
 provider "azurerm" {
-  version = ">= 1.1.0"
+  version = "= 1.21.0"
```

Can we use version >= `1.24.0` and < `2.0.0` ?
```suggestion
  version = "~> 1.2"
```